cargo/lib/dependabot/cargo/file_fetcher.rb

```diff
@@ -323,6 +324,15 @@ def cargo_lock
         @cargo_lock = fetch_file_if_present("Cargo.lock")
       end
 
+      def cargo_config
+        return @cargo_config if defined?(@cargo_config)
+
+        @cargo_config = fetch_support_file(".cargo/config")
+
+        @cargo_config ||= fetch_support_file(".cargo/config.toml")
+                          &.tap { |f| f.name = ".cargo/config" }
```

Comment:
Can you explain this bit? Can the config file be named `config` or `config.toml`?

Reply:
Cargo will read the `.cargo/config.toml` file, and fallback to the `.cargo/config` file as the `toml` file extension support was added in v1.39. This order here is backwards, so I am pushing up a change to fix that, but otherwise will be necessary to ensure that we pull in the file that Cargo expects even on older repositories.

Reply:
To add context here, Dependabot requires the file to be in `toml`-parsable format: 
* https://github.com/dependabot/dependabot-core/pull/7120

Are the `.cargo/config` files still in `toml` format but just lacking the file extension? Or are they a different format altogether?

Reply:
Same file format, it is just that Cargo did not originally require the extension.